src/Uno.UI/UI/Xaml/Controls/Popup/Popup.Android.cs

```diff
@@ -9,12 +9,16 @@
 using Windows.UI.Xaml.Controls.Primitives;
 using Windows.UI.Xaml.Media;
 using Uno.UI;
+using Android.Content;
+using Uno.Disposables;
 
 namespace Windows.UI.Xaml.Controls.Primitives
 {
 	public partial class Popup
 	{
+		private readonly SerialDisposable _disposable = new SerialDisposable();
 		private PopupWindow _popupWindow;
+		private Context _popupContext;
```

Comment:
This looks like a potential memory leak - I think you should be using WeakReference here.